nixos/modules/services/hardware/pid-fan-controller.nix

```
@@ -0,0 +1,190 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.pid-fan-controller;
+  heatSource = {
+    options = {
+      name = lib.mkOption {
+        type = lib.types.uniq lib.types.nonEmptyStr;
+        description = "Name of the heat source";
+      };
+      wildcardPath = lib.mkOption {
+        type = lib.types.nonEmptyStr;
+        description = ''
+          Path of the heat sources hwmon 'temp_input' file.
+          This path can contain multiple wildcards, but has to resolve to
+          exactly one result.
+        '';
+      };
+      pidParams = {
+        setPoint = lib.mkOption {
+          type = lib.types.ints.unsigned;
+          description = "Set point of the controller in °C";
```

```suggestion
          description = "The point of the controller in °C.";
```

I'm not exactly sure what `point` means here. If it's a technical term specific to PID, leave as it is, as it'll be obvious to users. If not, please explain it.

---

In control theory the set point is the value that should be reached by the system. Same goes for the term-gains K_*.

---

Ah, then it should be obvious to users. Just add the period and keep `Set` then.